### ISSUE TITLE
find 'cygdrive/j' : Permission denied

### ISSUE THREAD

**Issue by carc1n0gen** [babun/babun#282]
Upon installing a fresh babun 1.1.0 I have the following output:

```
Executing daily babun check:
Source consistent [OK]
Prompt speed      [OK]
File permissions  [OK]
Connection check  [OK]
Update check      [OUTDATED]
Hint: your babun is outdated. Execute 'babun update'
Cygwin check      [OK]
find: ‘/cygdrive/i’: Permission denied
find: ‘/cygdrive/j’: Permission denied
Welcome to babun - the Windows shell you will love!
You have got a lot of things under the hood here!

   pact -> a package manager that handles installation of babun packages
      pact install tar -> will install tar package on your system
      pact remove tar -> will remove tar package from your system

   babun -> a script that manages babun and enables auto-update
      babun check -> will check the configuration (proxy, connection, etc)
      babun update -> will update babun to the newest version
      babun --welcome -> will display this message again

There is a couple of plugins installed by default: oh-my-zsh, pact, cacert, etc.
There is also a lot of dev tools such as git, svn, unix utils, etc
If you would like to contribute, code up a plugin or report an issue just
go to github: https://github.com/babun/babun

If you like the project star it on github and follow me on twitter!
This project is created and maintained by Tom Bujok (@tombujok)
```

Note the `find 'cygdrive/j' : Permission denied` and `find 'cygdrive/i' : Permission denied`

Also when executing `babun update` as requested by this output I get:

```
Executing babun update
  upstream  [release]
  installed [1.1.0]
  newest    [1.1.1]
HEAD is now at 4c8eed3 Fixing version on fresh install
Fetching origin
error: Could not fetch origin
Error on or near line 47, last command 'git --git-dir="$babun/source/.git" --work-tree="$babun/source" fetch --all';
```

I then closed the terminal and opened a new one and now `babun update` outputs:

```
Executing babun update
: invalid option name: line 2: set: pipefail
```

**Comment by ylecuyer** [comment]
I have th same problem every time I launch babun

![babun](https://cloud.githubusercontent.com/assets/1866809/7084244/555942b0-df6b-11e4-8045-d680a87ed148.jpg)

**Comment by tombujok** [comment]
Could you execute the following commands and paste the result:
`find /cygdrive/ -maxdepth 1 -type d` 
and
`ls -al /cygdrive/`

**Comment by ylecuyer** [comment]
Here is the result:
![babun_2](https://cloud.githubusercontent.com/assets/1866809/7084424/8bc8f294-df6d-11e4-80d2-753644e3e535.jpg)

**Comment by tombujok** [comment]
Basically it looks like `u` is an unidentifed network drive which you don't have acces to.
Can you access `u` in a File Explorer?

**Comment by ylecuyer** [comment]
You're right. It's displayed in explorer but I can't access it.

**Comment by tombujok** [comment]
OK, it's just a stderr message from find.
Could you execute and past the result?
`find /cygdrive/ -maxdepth 1 -type d 2>/dev/null `

**Comment by ylecuyer** [comment]
You're right. It's displayed in explorer but I can't access it.

**Comment by tombujok** [comment]
I understand. It would be just great to check if redirecting the output to stderr removes this permission denied messages. 
`find /cygdrive/ -maxdepth 1 -type d 2>/dev/null`

**Comment by ylecuyer** [comment]
It does:
![babun_3](https://cloud.githubusercontent.com/assets/1866809/7089702/14599570-df9e-11e4-8cfb-b9f232a641ae.jpg)

**Comment by tombujok** [comment]
Perfect! Would you like to fix the second find execution in the file mentioned below and submit a pull request so that you get the credits ;) ?
https://github.com/babun/babun/blob/master/babun-core/plugins/cygdrive/start.sh

**Comment by carc1n0gen** [comment]
Any ideas about the babun update problem?

**Comment by tombujok** [comment]
Thanks!

**Comment by tombujok** [comment]
Fixed in 1.2.0
See release status: #304
